Make ::selection allowed-properties list match spec

hovdallakyan opened the pull request:
Replaced the list of the accepted properties to use with ::selection as described in spec 
https://drafts.csswg.org/css-pseudo-4/#highlight-styling

hovdallakyan committed:
Removed cursor property as accepted

Removed cursor property as accepted to use with ::selection.
According to the spec, cursor is not supposed to work https://drafts.csswg.org/css-pseudo-4/#highlight-styling.
It’s not strictly forbidden though, but it seems not the case, it doesn't work.

sideshowbarker commented on files/en-us/web/css/_doublecolon_selection/index.html:
```diff
@@ -23,7 +23,6 @@ <h2 id="Allowable_properties">Allowable properties</h2>
 <ul>
```
I notice that at https://drafts.csswg.org/css-pseudo-4/#highlight-styling the spec lists:

* color
* background-color
* text-decoration and its associated properties
* text-shadow
* stroke-color, fill-color, and stroke-width

So, along with `cursor`, as far as properties in this MDN article that aren’t in the spec, the spec lacks the following:

* caret-color
* outline
* text-emphasis-color

So, shouldn’t we remove those from the MDN article?

And as far properties that aren’t in this the MDN article, but are in the spec, the spec has the following:

* stroke-color, fill-color, and stroke-width

So, shouldn’t we add those?

In other words, shouldn’t we just replace the list that we currently have in the MDN with the following:

* color
* background-color
* text-decoration and its associated properties
* text-shadow
* stroke-color, fill-color, and stroke-width

HDallakyan replied:
totally agree

hovdallakyan committed:
Changed the accepted properties to use with ::selection

Replaced the list of the accepted properties to use with ::selection as described in spec https://drafts.csswg.org/css-pseudo-4/#highlight-styling

sideshowbarker commented:
Hovhannes, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉
